Content.Server/Shuttles/ShuttleDestinationCoordinatesComponent.cs

```diff
@@ -0,0 +1,13 @@
+
+namespace Content.Server.Shuttle.Components;
+
+/// <summary>
+/// Enables a shuttle to travel to a destination with an item inserted into its console
+/// </summary>
+[RegisterComponent]
+public sealed partial class ShuttleDestinationCoordinatesComponent : Component
+{
+    // Uid for entity containing the FTLDestination component
+    [DataField]
+    public EntityUid? Destination;
+}
```

Comment:
If you put it in shared you don't have to worry about any other networking for FTL disks once you update FTL code to handle it.